MinPlatformPkg/Test/Library/TestPointLib/MmTestPointCommunication.c

```diff
@@ -0,0 +1,378 @@
+/** @file
+
```

Comment:
Add brief file description for newly added file.

Reply:
Added.